boot/seal.go

```diff
@@ -54,6 +54,8 @@ var (
 // Hook functions setup by devicestate to support device-specific full
 // disk encryption implementations. The state must be locked when these
 // functions are called.
+//
+// XXX: move to "kernel/fde/fde.go" ?
```

Comment:
maybe,  the win of the current setup is that to run things in boot you need to mock things via boot. With this change to run things in boot you would need to mock things for kernel/fde